tests/cookbook/manual-sow-reap/input.dat

```diff
@@ -1,124 +0,0 @@
-#! Frequency by finite difference of energies.
-#! Automated version of this in pywrap-freq-e-sowreap. !nosample
```

Comment:
was this test not working right? it was the `energy(..., mode="sow")` that were definitely worth getting rid of. this might still be a useful model if it works.

Reply:
Currently not working. It seems `driver_findif.hessian_from_energies_geometries()` is generating displaced geometries as numpy arrays which `manual-sow-reap` is trying to JSON dump. Potentially an easy patch if this is worth keeping.

Reply:
@psi-rking, is it ok with you to drop the manual-sow-reap example file? I could maybe make it work, but it'd be awkward.

Reply:
@loriab, I think these are now irrelevant.  Their replacements are in psi4/samples/cookbook/manual-...
One of these (manual-fd-hess-grad) I just modified last week usefully - so I hope those are staying?

Reply:
Thanks. Yes, the manual-fd ones are staying.